packages/sqlserver/sodasql/dialects/sqlserver_dialect.py

```diff
@@ -66,6 +66,17 @@ def default_connection_properties(self, params: dict):
             'schema': 'public'
         }
 
+    def safe_connection_data(self):
+        return [
+            self.type,
+            self.host,
```

Comment:
we will push this into `datasource_id`

Reply:
`host` should be sufficient unless host is potentially often the same across different organisations.